packages/ui/src/components/molecules/RegionBar/styles.scss

```diff
@@ -5,7 +5,7 @@
 
   // Default properties
   --fs-region-bar-width                    : 100%;
-  --fs-region-bar-padding                  : var(--fs-spacing-0) var(--fs-spacing-1) var(--fs-spacing-1) var(--fs-spacing-2);
+  --fs-region-bar-padding                  : var(--fs-spacing-0) 0 var(--fs-spacing-0) var(--fs-spacing-2);
```

Comment:
Can we add @deprecated in the comments for 

// CTA 
// Message
// Postal Code